packages/xod-client/src/editor/containers/Patch.jsx

```diff
@@ -159,6 +183,14 @@ class Patch extends React.Component {
       }
     }
 
+    if (this.props.mode === EDITOR_MODE.PANNING) {
+      this.props.actions.setOffset(this.getOffset());
+
+      if (R.pathEq(['nativeEvent', 'which'], 2, event)) {
```

Comment:
I’d extract `isMiddleButtonPressed :: Event -> Boolean` as it much more clear than when used directly inline.